otherlibs/action-plugin/test/one-undeclared-target/dune

```diff
@@ -0,0 +1,10 @@
+(cram
+ (alias run_dynamic)
+ (deps
+  (package dune)
+  (glob_files bin/*.exe)))
+
+(alias
+ (name runtest)
+ (deps
+  (alias run_dynamic)))
```

Comment:
I'm wondering if we shouldn't allow `(aliases runtest run_dynamic)`. The above seems to be a common pattern.

Reply:
Actually, the extra `alias` could just be removed. `runtest` is already added automatically, it's just that I forgot to change this from a previous iteration on the stanza.